crates/modelardb_embedded/bindings/python/docs/conf.py

```diff
@@ -0,0 +1,30 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
General question to all the files added in this PR. Should they all have the Copyright header? Maybe it is only necessary for the larger files and not the individual `rst` files.

Reply:
All files should now have copyright headers, @chrthomsen, please double-check and resolve the conversation if I got all of them.